packages/core/src/Stores/client-store.js

```diff
@@ -494,6 +496,33 @@ export default class ClientStore extends BaseStore {
         });
     }
 
+    @action.bound
+    getSelfExclusion() {
+        return new Promise(resolve => {
+            WS.authorized.storage.getSelfExclusion().then(data => {
+                runInAction(() => {
+                    if (data.get_self_exclusion) {
+                        this.self_exclusion = data.get_self_exclusion;
+                    } else {
+                        this.self_exclusion = false;
+                    }
+                    resolve(data);
+                });
+            });
+        });
+    }
```

Comment:
Nit, add an extra space after this function to stay consistent with the rest of the file.